gaphor/SysML/propertypages.py

```diff
@@ -206,24 +215,43 @@ def _on_item_flow_active(self, switch, gparam):
             subject.informationFlow = iflow
             iflow.informationSource = subject.end[0].role
             iflow.informationTarget = subject.end[1].role
-            iflow.itemProperty = subject.model.create(sysml.Property)
+            iflow.itemProperty = subject.model.create(UML.Property)
```

Comment:
Why the UML Property instead of the sysml one?

Reply:
It's the same. `sysml` imports `Property` from `gaphor.UML`

Reply:
Ya, I get that they are the same. It just seems clearer to me that the SysML profile uses SysML elements.